utils/stdx/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "stdx"
+version = "0.0.0"
+authors.workspace = true
+edition.workspace = true
+publish = false
+
+[dependencies]
+# Must not depend on any crates from nearcore workspace.
```

Comment:
```suggestion
[dependencies]
# Absolutely must not depend on any crates from nearcore workspace,
# and should have as few dependencies as possible otherwise. 
```